apps/condo/domains/meter/utils/serverSchema/resolveHelpers.js

```diff
@@ -0,0 +1,44 @@
+const get = require('lodash/get')
+const { Contact } = require('@condo/domains/contact/utils/serverSchema')
+const { Meter } = require('@condo/domains/meter/utils/serverSchema')
+
+async function addClientInfoToResidentMeterReading (context, resolvedData) {
```

Comment:
helpers need to be covered by tests too :)

Reply:
i rewrote the `resident: can create MeterReadings in his unit` test and checked in it that client info added to `MeterReading`

Reply:
seems like it's covered in meterReading test

Reply:
moved `resident: client info saved in new reading` into a separate test